code/game/turfs/unsimulated/floor.dm

```diff
@@ -19,3 +19,80 @@
 
```

Comment:
Shouldn't these turfs be in the persistence modpack? Or do you plan on pushing them upstream

Reply:
I could push them upstream yeah